srcpkgs/marvin/template

```diff
@@ -0,0 +1,56 @@
+# Template for Marvin
+pkgname=marvin
+version=18.16.0
+revision=1
+maintainer="Brenton Horne <brentonhorne77@gmail.com>"
+homepage="https://chemaxon.com/products/marvin"
+license="marvin"
+depends="oracle-jdk"
+short_desc="Skeletal structure drawing program by ChemAxon"
+only_for_archs="x86_64"
+repository=nonfree
+restricted=yes
+nostrip=yes
+
+_baseUrl="https://dl.chemaxon.com/marvin/"
+_filename="marvin_linux_$(echo $version | sed 's/.[0-9]*$//g').deb"
+_marvinUrl="https://dl.chemaxon.com/marvin/${version}/${_filename}"
+distfiles="${_marvinUrl}"
+checksum="adfa5917d59a277732642a7771a763abf7f5617ff551320b203602d73d3abbcf"
+
+do_extract() {
+	mkdir -p ${DESTDIR}
+	ar x ${XBPS_SRCDISTDIR}/${pkgname}-${version}/${_filename}
+}
+
+do_install() {
+	tar xf data.tar.gz -C ${DESTDIR}
```

Comment:
move this line (without `-C ${DESTDIR}`) in `do_extract()` just before `ar x ...`

Reply:
Just *after* don't you mean? `ar x` is needed to open the deb file, before it's run you've got no tarball to decompress.

Reply:
yes, sorry ;) just after.

Reply:
I got this precise order from the [google-chrome template](https://github.com/void-linux/void-packages/blob/4efc2df55d08261609232762e42aa3f3b3ad5e22#diff-eeac49eef1932c314926f4e9b8c0bbda/srcpkgs/google-chrome/template#L31), so as a follow up PR it might be nice to fix that order too.

Reply:
404 page not found.

Reply:
Oh it seems you were wrong here. I made the change you wanted and I ended up with a package that was 629B in size. It seems the whole /opt directory wasn't being included in the end package. It seems tar xf needs to be in the `do_install()` function, otherwise /opt doesn't end up in the finished package. So I'm going to revert this.

Reply:
Just copy the files in `/opt` during `do_install()` ...

Reply:
My apologies, although I have to ask where does tar xf extract the file to when called in `do_extract()`? `${XBPS_SRCDISTDIR}`?

Reply:
Ah sorry, thanks for your help and I have now implemented it.